Signal/src/ViewControllers/DebugUI/DebugUISessionState.m

```diff
@@ -20,63 +20,112 @@ - (NSString *)name
 
 - (nullable OWSTableSection *)sectionForThread:(nullable TSThread *)threadParameter
```

Comment:
Modified `DebugUISessionState` to have per-thread and no-thread modes, like the other Debug UI pages.